viewer/index.html

```diff
@@ -100,6 +100,7 @@
                 url: new URL("https://github.com/NYPL-Simplified/webpub-viewer"),
                 label: "Return to Digital Research Books Beta",
                 ariaLabel: "Return to Digital Research Books  Beta",
+                libraryIcon: new URL("https://www.nypl.org/sites/default/files/images/av/news_2009_11_06_logo.jpeg")
```

Comment:
It's weird to me that you now have to pass in a jpeg for the library logo?  If it defaults to the svg when nothing is passed in, then let's just not pass anything in.  It's either that, or allow the libraryIcon to be passed as an svg.

Reply:
Now it's setup so if a `libraryIcon`  is not passed in none of this information will display. I can update it to just send am empty string which will then display the default SVG. Before I was leaning towards libraryIcon being an optional value.

Another option is to add logic to treat SVGs differently from other types of files but adding SVG support would be more involved since the SVG would need to be edited to include attributes such as:
```
width = "${IconLib.WIDTH_ATTR}" height = "${IconLib.HEIGHT_ATTR}" viewBox = "${IconLib.VIEWBOX_ATTR} "aria-labelledby="up-label" 
```
I will check to see if we are expecting Open EBooks logo to be an SVG...

Reply:
Let's do the simplest thing for now.  

I can imagine people wanting to do three things:  

1) I am a part of NYPL, I want to keep the NYPL icon but link to a different site (should use the SVG icon.  I think, best thing to do here is to accept 
```
upLink: {
          url: new URL("http://up.com"),
          label: "Up Text",
          ariaLabel: "Up Aria Text",
        }
```
and put the icon as the fallback svg
2) I am not a part of NYPL, I want to pass in my own logo and link.  I would pass in 
```
upLink: {
          url: new URL("http://up.com"),
          label: "Up Text",
          ariaLabel: "Up Aria Text",
          libraryIcon: new URL("icon.png") 
        }
```
3) I don't care, and don't want to configure this.  
Either, don't pass in upLink at all 
```
IFrameNavigator.create({
        element,
        manifestUrl: new URL("http://example.com/manifest.json"),
        store,
        settings,
        annotator,
        publisher,
        serif,
        sans,
        day,
        sepia,
        night,
        paginator,
        scroller,
        eventHandler,
      });
```
and it defaults to the library svg with placeholder link/label/arialabel,

or, still require something like this

```
upLink: {
          url: new URL("http://up.com"),
          label: "Up Text",
          ariaLabel: "Up Aria Text",
        }
```
to be passed in